.github/workflows/build-msvc.yml

```diff
@@ -101,7 +101,7 @@ jobs:
               || failed=$?
             if ((failed)) ; then cat config.log ; exit $failed ; fi
           fi
-          make -j || failed=$?
+          make -j$(nproc) || failed=$?
```

Comment:
Assuming this one hasn't failed before, though, can this one be left as-is (my concern is that we unexpectedly make this slower, without actually fixing anything - I'm less worried in the build-cross workflow about it being slower, because that workflow is only executed on-demand)

Reply:
Left as-is.